doc/bc_election_turnout_report.toc

```diff
@@ -0,0 +1,7 @@
+\contentsline {subsection}{Summary}{1}{section*.2}%
```

Comment:
Is this file autogenerated, or did you add it manually?

Reply:
It looks like it was auto generated.